packages/web-react/src/components/Stack/demo/StackBlocksWithInnerDividersAndVerticalSpacing.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import DocsBox from '../../../../docs/DocsBox';
+import Stack from '../Stack';
+
+const StackBlocksWithInnerDividersAndVerticalSpacing = () => (
+  <Stack elementType="ul" hasSpacing hasIntermediateDividers>
+    {[1, 2, 3].map((i) => (
```

Comment:
:-) Generating values is an interesting problem. There are multiple ways/oneliners how to, based on a single value, create an array full of values.

```js
var foo = [ ...Array(N).keys() ];
var foo = Array(N).fill().map((v,i)=>i);
Array.from({ length: N }, (v, i) => i)
```

;-)

Reply:
And which one is the best? :)

Reply:
All are good! 😁